### ISSUE TITLE
css stylesheet is Missing

### ISSUE THREAD

**Issue by nayeem81127** [salesforce-ux/design-system#715]
C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:13231
      var error = new self.Error(message);
                  ^

Error: Can't find stylesheet to import.
   ╷
14 │   '../design-tokens/dist/theme-one-salesforce.default',
   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  ui\_design-tokens.scss 14:3       @import
  ui\_init.scss 11:9                @import
  ui\customizer-erp_mark7.scss 7:9  root stylesheet
    at Object._newRenderError (C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:13231:19)
    at Object._wrapException (C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:13059:16)
    at StaticClosure._renderSync (C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:13037:18)
    at Object.Primitives_applyFunction (C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:1136:30)
    at Object.Function_apply (C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:5903:16)
    at _callDartFunctionFast (C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:7605:16)
    at Object.renderSync (C:\Users\Downloads\design-system\node_modules\sass\sass.dart.js:7583:18)
    at C:\Users\Downloads\design-system\tools\customizer.cli.js:127:26 { 
  formatted: "Error: Can't find stylesheet to import.\n" +
    '   ╷\n' +
    "14 │   '../design-tokens/dist/theme-one-salesforce.default',\r\n" +
    '   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n' +
    '   ╵\n' +
    '  ui\\_design-tokens.scss 14:3       @import\n' +
    '  ui\\_init.scss 11:9                @import\n' +
    '  ui\\customizer-erp_mark7.scss 7:9  root stylesheet',
  line: 14,
  column: 3,
  file: 'C:\\Users\\Downloads\\New folder\\design-system\\ui\\_design-tokens.scss',  status: 1
}

**Comment by owenschoppe** [comment]
@nayeem81127 Can you clarify what you are trying to do?

**Comment by laurensuh** [comment]
Hi @nayeem81127 Are you still experiencing this issue? If you can give us more context that would be great. If this is no longer an issue, let us know and we can close this for you. Thanks!

**Comment by mmatsumoto-ts** [comment]
Hi @laurensuh. I am getting the same error.
Occurs when running the command  `npm run slds: customizer` on this [site](https://www.lightningdesignsystem.com/tools/scope-customizer/).

```
➜ Processing...

(node:14644) UnhandledPromiseRejectionWarning: Error: Can't find stylesheet to import.
   ╷
14 │   '../design-tokens/dist/theme-one-salesforce.default',
   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  ui\_design-tokens.scss 14:3  @import
  ui\_init.scss 11:9           @import
  ui\customizer-test.scss 8:9  root stylesheet
    at Object._newRenderError (C:\slds-rep\design-system\node_modules\sass\sass.dart.js:13231:19)
    at Object._wrapException (C:\slds-rep\design-system\node_modules\sass\sass.dart.js:13059:16)
    at StaticClosure._renderSync (C:\slds-rep\design-system\node_modules\sass\sass.dart.js:13037:18)
    at Object.Primitives_applyFunction (C:\slds-rep\design-system\node_modules\sass\sass.dart.js:1136:30)
    at Object.Function_apply (C:\slds-rep\design-system\node_modules\sass\sass.dart.js:5903:16)
    at _callDartFunctionFast (C:\slds-rep\design-system\node_modules\sass\sass.dart.js:7605:16)
    at Object.renderSync (C:\slds-rep\design-system\node_modules\sass\sass.dart.js:7583:18)
    at C:\slds-rep\design-system\tools\customizer.cli.js:127:26
(node:14644) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:14644) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Thank you.

**Comment by kflournoy-sf** [comment]
@mmatsumoto-ts Thanks for giving us some more information, we'll take a look and see if we can figure out what's going on!

**Comment by sfboss** [comment]
For what it's worth as a shot in the dark I ran 'npm run gunk -- build' facing this same issue, and it seemed to have generated what that command would then be looking for and I was able to generate normally after.  I was just looking at what was available to run and it was listed for the "build" command and sounded like something needed built.

I'm mid-troubleshooting trying to even use it,  and admittedly novice in node and it's nuances, but the SLDS sites instructions mention to do 'npm run slds:customizer' and the prework needed didn't seem to have been done yet after a clone of the repo and a cd into the directory and then 'npm install', per their first step instructions.

When I went to originally run the npm install in the folder I got a message about a dependency check process it was going to run and it grabbed a large amount of new packages before ultimately saying it had installed.   Mentioning this in case some periodic node update of modules I could be unaware of could be a symptom.

Hope that helps anyone
